responsibleai/responsibleai/_tools/causal/causal_result.py

```diff
@@ -86,9 +85,97 @@ def _whatif(self, X, X_feature_new, feature_name, y, alpha=0.1):
         return self.causal_analysis.whatif(
             X, X_feature_new, feature_name, y, alpha=alpha)
 
-    def _cohort_effects(self, X_test, alpha=0.01, keep_all_levels=False):
-        return self.causal_analysis.cohort_causal_effect(
-            X_test, alpha=alpha, keep_all_levels=keep_all_levels)
+    def _global_cohort_effects(self, X_test):
+        causal_data = CausalData()
+
+        causal_data.id = self.id
+        causal_data.version = self.version
+        causal_data.config = self._get_config_object(self.config)
+        causal_data.global_effects = \
+            self.causal_analysis.cohort_causal_effect(
+                X_test, alpha=self.config.alpha,
+                keep_all_levels=True).to_dict(
+                    orient="records")
+        return causal_data
+
+    def _local_instance_effects(self, X_test):
+        causal_data = CausalData()
+
+        causal_data.id = self.id
+        causal_data.version = self.version
+        causal_data.config = self._get_config_object(self.config)
```

Comment:
nit: refactor out the part to create the CausalData object with id, version and config into a separate method called from both this function and _global_cohort_effects and _global_cohort_policy methods, they all seem to have the exact same logic